### PR TITLE
Compatibility with vedo 2023.5.0

### PR DESCRIPTION
**Pull request by IgorTatarnikov**
Before submitting a pull request (PR), please read the [contributing guide](https://github.com/brainglobe/.github/blob/main/CONTRIBUTING.md).

## Description

**What is this PR**

This PR addresses any changed to the API made in vedo 2023.5.0.

- [ ] Bug fix
- [ ] Addition of a new feature
- [x] Other

**Why is this PR needed?**

vedo 2023.5.0 was released.

## References

#261 

## How has this PR been tested?

All tests pass locally, all examples apart from `examples/streamlines.py` run.

## Is this a breaking change?

If this PR breaks any existing functionality, please explain how and why.

## Checklist:

- [x] The code has been tested locally
- [x] The code has been formatted with [pre-commit](https://pre-commit.com/)

### CODE REVIEW

**Review by alessandrofelder — APPROVED** [review]
Looks good - couple of tiny comments.

As I presume you've seen, the new way to set the color mode (?) can be gleaned from [this vedo example](https://github.com/marcomusy/vedo/commit/7a0157503fb6198859cdddd34ae17e60d7fc3c65#diff-161d7efffe096772e05a48076069c4cfb31a4f3b51a6b762865fbfe9c522d668) - no idea whether we want to/need/should support this (@adamltyson ?)

maybe open a low priority issue so we don't forget? Unless you already had another plan around how to fix this, @IgorTatarnikov ?

File: brainrender/actors/ruler.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def ruler(p1, p2, unit_scale=1, units=None, s=50):
     dist = mag(p2 - p1) * unit_scale
     label = precision(dist, 3) + " " + units
     lbl = Text3D(label, pos=midpoint, s=s + 100, justify="center")
-    lbl.SetOrientation([0, 0, 180])
+    # lbl.SetOrientation([0, 0, 180])
```
[the remark this answers is not in the document]
Comment (IgorTatarnikov) [reply brainrender/actors/ruler.py:40]: I only left it as a sort of reminder that this used to do something and now it doesn't. The labels aren't in the right place when the `add_labels.py` example is run so I thought that'll be a good first place to look when we get to fixing it.

File: examples/neurons.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 scene = Scene(title="neurons")
 
 # Add a neuron from file
-scene.add(Neuron("examples/data/neuron1.swc"))
+scene.add(Neuron("data/neuron1.swc"))
```
[the remark this answers is not in the document]
Comment (IgorTatarnikov) [reply examples/neurons.py:16]: I can definitely make an issue, most paths need to be cleaned up so they're not as fragile

File: examples/screenshot.py [hunk 3]
```diff
@@ -32,7 +32,7 @@
     "focalPoint": (7718, 4290, -3507),
     "distance": 40610,
 }
-zoom = 1.5
+zoom = 2.5
```
[the remark this answers is not in the document]
Comment (IgorTatarnikov) [reply examples/screenshot.py:35]: I honestly just took it from [here](https://github.com/brainglobe/brainrender/issues/261#issuecomment-1787970631) assuming that something changed with how the zoom is calculated in vedo.

File: examples/user_volumetric_data.py [hunk 4]
```diff
@@ -74,7 +74,8 @@
 
 # 3. create a Volume vedo actor and smooth
 print("Creating volume")
-vol = Volume(transformed_stack, origin=scene.root.origin()).smooth_median()
+vol = Volume(transformed_stack).permute_axes(2, 1, 0)
```
[the remark this answers is not in the document]
Comment (IgorTatarnikov) [reply examples/user_volumetric_data.py:77]: I'm moving that out of the examples and into the core package, I've left some of the post compatibility changes off of this PR.

**Comment by adamltyson** [comment]
> As I presume you've seen, the new way to set the color mode (?) can be gleaned from [this vedo example](https://github.com/marcomusy/vedo/commit/7a0157503fb6198859cdddd34ae17e60d7fc3c65#diff-161d7efffe096772e05a48076069c4cfb31a4f3b51a6b762865fbfe9c522d668) - no idea whether we want to/need/should support this (@adamltyson ?)

No idea tbh

**Comment by alessandrofelder** [comment]
> No idea tbh

I'd suggest leaving it out then until a user asks for the functionality, but also happy with an additional `mode` argument for  if we wanna support it - no strong opinion. This can be merged once you're happy with it @IgorTatarnikov - I am.

**Pull request merged**